packages/node-config-provider/src/fromStatic.ts

```diff
@@ -0,0 +1,13 @@
+import { fromStatic as convertToProvider } from "@aws-sdk/property-provider";
+import { Provider } from "@aws-sdk/types";
+
+export type FromStaticConfig<T> = T | (() => T);
+type Getter<T> = () => T;
+const isFunction = <T>(func: FromStaticConfig<T>): func is Getter<T> => typeof func === "function";
+
+export const fromStatic = <T>(defaultValue: FromStaticConfig<T>): Provider<T> =>
+  isFunction(defaultValue)
+    ? async () => {
+        return defaultValue();
+      }
```

Comment:
```suggestion
    ? async () => defaultValue()
```

Reply:
Fixed in [`61cffe9` (#1420)](https://github.com/aws/aws-sdk-js-v3/pull/1420/commits/61cffe929238596542ba351bef8caf739dd76ce7)